src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -507,7 +575,9 @@ static int do_map()
   common_init_finish(g_ceph_context);
   global_init_chdir(g_ceph_context);
 
-  if (socketpair(AF_UNIX, SOCK_STREAM, 0, fd) == -1) {
+  bool b = socketpair(fd);
```

Comment:
Nit: either rename ```b``` to something meaningful or just put the ```socketpair``` call back into the ```if``` statement and avoid the extra variable.